CharlieBackend.Api/Controllers/EventsController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CharlieBackend.Business.Services.Interfaces;
+using CharlieBackend.Core;
+using CharlieBackend.Core.DTO.Schedule;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace CharlieBackend.Api.Controllers
+{
+    /// <summary>
+    /// Controller to manage scheduled events data
+    /// </summary>
+    [Route("api/events")]
+    [ApiController]
+    public class EventsController : ControllerBase
+    {
+        private readonly IEventsService _eventsService;
+
+
+        /// <summary>
+        /// Events controller constructor
+        /// </summary>
+        /// <param name="eventsService"></param>
+        public EventsController(IEventsService eventsService)
+        {
+            _eventsService = eventsService;
+        }
+
+        /// <summary>
+        /// Get event by id
+        /// </summary>
+        /// <response code = "200" > Successful getting of schedule</response>
+        /// <response code="HTTP: 404">Error, given scheduled event not found</response>
+        [SwaggerResponse(200, type: typeof(ScheduledEventDTO))]
+        [Authorize(Roles = "Secretary, Admin")]
+        [HttpGet("{id}")]
+        public async Task<ActionResult<ScheduledEventDTO>> GetConcreteScheduleByID(long id)
```

Comment:
Please rename controllers methods to DeleteEvent, UpdateEvent

Reply:
Done